metadata.yml

```diff
@@ -2,7 +2,7 @@ rules:
   - apiGroups: ["admissionregistration.k8s.io"]
     apiVersions: ["v1"]
     resources:
-      ["validatingwebhookconfigurations", "mutatingwebhookconfigurations"]
+    lidatingwebhookconfigurations", "mutatingwebhookconfigurations"]
```

Comment:
```suggestion
      ["validatingwebhookconfigurations", "mutatingwebhookconfigurations"]
```

Reply:
How does this even happen?????

Reply:
@olblak have you ever seen this (wrong) change, from an updatecli YAML target?
Specifically, this one: https://github.com/kubewarden/automation/blob/main/updatecli/policies/open-release-pr/updatecli.d/open-release-pr.yaml#L46-L56